drv/sidecar-front-io/src/leds.rs

```diff
@@ -1,11 +1,15 @@
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
+use super::transceivers::{LogicalPort, LogicalPortMask};
 use drv_i2c_api::I2cDevice;
 use drv_i2c_devices::pca9956b::{Error, LedErr, Pca9956B};
+use drv_transceivers_api::NUM_PORTS;
 
 pub struct Leds {
     controllers: [Pca9956B; 2],
+    current: u8,
```

Comment:
We could use a docstring here about `current` and `pwm` units

Reply:
Good recommendation. I'd had this below near the `DEFAULT_LED_{CURRENT,PWM}` constants for this, but in this PR I made them programmable so it makes sense to pull that text up to this struct.